techmatters/hrm
# chore: Removed deprecated case update endpoint (and deps)

## Description
This PR removes the marked-for-deprecation case `update` endpoint `casesRouter.put('/:id')`, and all of it's downstream dependencies.

### Checklist

### Verification steps
Sanity check to confirm everything works as intended with newer versions of Flex.

### AFTER YOU MERGE

1. Cut a release tag using the GitHub workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P
